lib/timetable-form/Form.tsx

```diff
@@ -113,6 +139,22 @@ export const Form = (props: React.HTMLAttributes<HTMLDivElement>) => {
       <h1 className="govuk-heading-xl" data-testid="form-title">
         Timetable Form
       </h1>
+      {formErrors.length > 0 && (
+        <div className="govuk-error-summary" data-module="govuk-error-summary">
+          <div role="alert">
+            <h2 className="govuk-error-summary__title">There is a problem</h2>
+            <div className="govuk-error-summary__body">
+              <ul className="govuk-list govuk-error-summary__list">
+                {formErrors.map((error) => (
+                  <li>
+                    <a href={`#${error.path.join("-")}`}>{error.message}</a>
+                  </li>
+                ))}
+              </ul>
+            </div>
+          </div>
+        </div>
+      )}
```

Comment:
Maybe something to look at later with design but potentially, this should live at the bottom of the form, below the validate button. At the moment, you need to manually scroll back up if there are errors.

Another thing to think about though is that eventually this will be multiple pages, so this will be less of an issue. This will likely also answer the question of when we validate because we'd have a continue button or similar.

Reply:
I have placed it at the top of the page as that seems to be what [GDS](https://design-system.service.gov.uk/components/error-summary/) recommends

Reply:
No problem, I think GDS also recommend one question per page, so we can tackle this later